installer_instructions/INSTALLER.md

```diff
@@ -0,0 +1,46 @@
+## Installer instructions
+
+DAppNode releases are triggered through the [DNP_CORE](https://github.com/dappnode/DNP_CORE) package, when new core packages are released the DNP_CORE [dependencies](https://github.com/dappnode/DNP_CORE/blob/fc0891669870407f3673a8a6273df780efadd902/dappnode_package.json#L23) are updated with the new versions. Within core packages there are **mandatory** core packages and **non-mandatory** core packages. Both of them are part of the system packages category and perform critical action in DAppNode, such as connectivity methods, IPFS hash server for packages hashes, DNS resolver for dappnode domains and more.
+
+- Mandatory: [BIND](https://github.com/dappnode/DNP_BIND), [DAPPMANAGER](https://github.com/dappnode/DNP_DAPPMANAGER), [IPFS](https://github.com/dappnode/DNP_IPFS) and [WIFI](https://github.com/dappnode/DNP_WIFI)
```

Comment:
How is wireguard not mandatory?  And why is Wi-Fi mandatory. Cloud/server distros shouldn't have Wi-Fi at all. It only works on some NUCs so far from my experience testing newer machines etc.  there's issues with NUC11s as I recently found and assume there to be more with that in the new NUC12s. Is Wi-Fi included as a mandatory core package because the local network proxy (which for me as always been buggy since day 1) is built into the Wi-Fi package?

Reply:
That is what are system requirements now. You cannot not install WIFI for example. I agree that should be discussed and changed.

Reply:
resolved the wireguard install conditions already